src/esp/bindings/AttributesBindings.cpp

```diff
@@ -242,10 +242,6 @@ void initAttributesBindings(py::module& m) {
           &StageAttributes::setHouseFilename,
           R"(Handle for file containing semantic type maps and hierarchy for
           constructions built from this template.)")
-      .def_property(
-          "light_setup", &StageAttributes::getLightSetup,
-          &StageAttributes::setLightSetup,
-          R"(Habitat lighting setup to use for constructions built by this template.)")
```

Comment:
This has been removed from the bindings because we will no longer support the user specifying the lighting setup they wish to use via stage attributes, but rather through simulator configuration or scene instance attributes. We leave the references in the stage attributes, however, so that the desired lighting config can be specified and passed internally during stage loading.